### DISCUSSION TITLE
Utilizar react-query

### DISCUSSION THREAD

**Discussion by Rodrigo001-dev** [diego3g/rsxp-2023#25] [General]
O que vocês acham de utilizar o react-query nesse projeto?

**Comment by carlosdavid0** [comment]
Indispensável.

**Comment by diego3g** [comment]
Com certeza! <3 

Vou fechar essa discussão porque estamos com bastante discussões abertas com tópicos que super valem a contribuição de vocês por lá também e acho que não vale estender muito a discussão por aqui. Sintam-se livres para reabrir a discussão se fizer sentido 🤗
